audb/core/api.py

```diff
@@ -93,20 +93,52 @@ def cached(
         name: str = None,
         shared: bool = False,
 ) -> pd.DataFrame:
-    r"""List available databases in the cache.
+    r"""List available databases and flavors in the cache.
 
     Args:
         cache_root: cache folder where databases are stored.
             If not set :meth:`audb.default_cache_root` is used
         name: name of database.
             If provided,
             it will show only cached versions of that database
-        shared: list shared databases
+        shared: include databases from shared cache
 
     Returns:
         cached databases
+        with cache path as index,
+        and name,
+        flavor_id,
+        version,
+        complete,
+        bit_depth,
+        channels,
+        format,
+        mixdown,
+        sampling_rate
+        as columns
 
-    """
+    Example:
+        >>> db = audb.load(
+        ...     'emodb',
+        ...     version='1.1.1',
+        ...     only_metadata=True,
+        ...     full_path=False,
+        ...     verbose=False,
+        ... )
+        >>> df = cached()
+        >>> for column in df.columns:
```

Comment:
Maybe replace with:

```
    Example:
        >>> db = audb.load(
        ...     'emodb',
        ...     version='1.1.1',
        ...     only_metadata=True,
        ...     full_path=False,
        ...     verbose=False,
        ... )
        >>> df = cached()
        >>> print(df.iloc[0].to_string())
        name                emodb
        flavor_id        d3b62a9b
        version             1.1.1
        complete            False
        bit_depth            None
        channels             None
        format               None
        mixdown             False
        sampling_rate        None
```

Reply:
Cool, I was sue there must be a better way, but I never can remember the `pandas` stuff ;)
I changed accordingly.